codex-ui/dev/pages/layout/Navbar.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="navbar-wrapper">
+    <Navbar>
+      Left
+      <template #navbar__right>
+        Right
+      </template>
+    </Navbar>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Navbar } from '../../../src/vue';
+</script>
+
+<style scoped>
+.navbar-wrapper {
+    min-height: 400px;
+    position: relative;
+    background-color: var(--base--bg-secondary);
+    border-radius: var(--radius-m);
+    border: 1px solid var(--base--border);
+    overflow-x: clip;
```

Comment:
is it used?

Reply:
yes, for demo

Reply:
I'm not sure that it is used. Explain what will happen without it.